vuepress/docs/next/docs/reference/glossary.md

```diff
@@ -0,0 +1,75 @@
+| Entando Web Content Management System (WCMS) | Entando's web content management system |
+| Local Hub | The local component repository in an Entando App Builder |
+| Entando Platform Capability (EPC) | A packaged capability that adds functionality to the platform and/or additional UX controls to the App Builder |
+| Entando Portal UI | 
+| Kubernetes integration service | Provides an abstraction layer between Entando microservices and the APIs exposed by a Kubernetes cluster |
+
+## General Technical Terms
+
+| Term |  Description
+|:--|:--
+| backend for frontend (BFF) | A software architecture pattern that connects the frontend of an application with the backend microservices |
+| component | An application building block such as a page template, content template, UX Fragment, widget, micro frontend, microservice, etc. |
+| component collection | A packaged set of single components that are in some way functionally unrelated |
+| content | The information/data contained within a component |
+| content type | The structure of specific content, defined by a set of content attributes |
+| content attribute | The basic data that define a content type |
+| content template | The style or layout of a content type where a single content type can have multiple content models defining different ways to render the same content |
+| digital asset | Images, documents, and other media files supported in a specific format |
+| low-code | A software development approach that leverages visual interfaces with basic logic and drag-and-drop capabilities to build applications and processes |
+| micro frontend (MFE) | A web development approach that implements an architecture characterized by a composition of compact frontend applications |
+| microservice | A software development technique characterized by an architecture of loosely coupled, granular services that employ lightweight protocols |
+| module | A basic element partitioned into a system of building blocks (e.g. PBCs, components, bundles, templates) that are independent, reusable, and can easily be configured into complex and useful structures |
+| no-code | A programming platform on which applications are built exclusively using a visual development interface to drag and drop software components |
+| pro-code | The traditional coding technique that relies on complex programming languages to build products such as websites and software |
+| site template | A bundle or collection of bundles that encompasses the functionality to bootstrap development of an Entando Application. A site template does not contain capabilities focused on specific business requirements but can be contained within an Entando solution template. |
+| solution template | A pre-packaged set of PBCs, component collections and single components providing a full-featured, domain specific solution |
+| UX fragment | An HTML block containing Freemarker tags that allow content to be rendered dynamically according to context |
```

Comment:
Move UX Fragment to Entando. I'm leaning towards keeping the Content*4 items to Entando as well.